v20.1/time.md

```diff
@@ -1,39 +1,60 @@
+
+    [Like Postgres](https://www.postgresql.org/docs/current/static/datatype-datetime.html), we implement the `TIMETZ` variant for [SQL standards compliance](sql-feature-support.html). We also implement the `TIMETZ` variant for compatibility with ORMs, like [Hibernate](build-a-java-app-with-cockroachdb-hibernate.html).
+
+{{site.data.alerts.callout_success}}
+We recommend always using `TIME` instead of `TIMETZ`. To use time zone UTC offsets, convert UTC values to the appropriate time zone on the client side.
+{{site.data.alerts.end}}
+
 ## Aliases
 
 In CockroachDB, the following are aliases:
 
-`TIME WITHOUT TIME ZONE`
+Alias    | Long Version
+---------|-------------
+`TIME`   | `TIME WITHOUT TIME ZONE`
+`TIMETZ` | `TIME WITH TIME ZONE`
 
 ## Syntax
 
-A constant value of type `TIME` can be expressed using an
-[interpreted literal](sql-constants.html#interpreted-literals), or a
-string literal
-[annotated with](scalar-expressions.html#explicitly-typed-expressions)
-type `TIME` or
-[coerced to](scalar-expressions.html#explicit-type-coercions) type
-`TIME`.
+A constant value of type `TIME` can be expressed using an [interpreted literal](sql-constants.html#interpreted-literals), or a string literal [annotated with](scalar-expressions.html#explicitly-typed-expressions) type `TIME` or  [coerced to](scalar-expressions.html#explicit-type-coercions) type `TIME`. When it is unambiguous, a simple unannotated [string literal](sql-constants.html#string-literals) can also be automatically interpreted as type `TIME`.
+
+The string format for `TIME` is `HH:MM:SS.SSSSSS`. For example: `TIME '05:40:00.000001'`. The fractional portion is optional and is rounded to microseconds (i.e., six digits after the decimal) for compatibility with the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/static/protocol.html).
 
-The string format for time is `HH:MM:SS.SSSSSS`. For example: `TIME '05:40:00.000001'`.
+To express a `TIMETZ` value (with a time zone offset from UTC), simply add an offset to a `TIME` value. For example, `TIMETZ '10:10:10.555555-05:00'` offsets from UTC by -5.
 
-When it is unambiguous, a simple unannotated [string literal](sql-constants.html#string-literals) can also
-be automatically interpreted as type `TIME`.
+If no time zone is specified for `TIMETZ`, the `timezone` [session variable](show-vars.html#supported-variables) is used. For example, if you [set the `timezone`](set-vars.html#set-time-zone) for a session using `SET TIME ZONE -2`, and you define the `TIMETZ` as `TIMETZ '10:10:10.55'`, the value will be displayed with an offset of -2 from UTC.
 
-Note that the fractional portion of `TIME` is optional and is rounded to microseconds (i.e., six digits after the decimal) for compatibility with the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/static/protocol.html).
+{{site.data.alerts.callout_info}}
+A date of `0000-01-01` is displayed for all `TIME`/`TIMETZ` values, but is not stored in the database.
```

Comment:
I would add for display, casting it to `string` will print properly, e.g. `SELECT '11:00'::time::string`.
Also, `'24:00'::timetz` displays as `0000-01-01 00:00:00` but is actually  `24:00:00` underneath and only a `::string` will catch that.